app/frontend/src/wells/summary.js

```diff
@@ -0,0 +1,37 @@
+/*
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+import 'babel-polyfill'
+import Vue from 'vue'
+import Vuex from 'vuex'
+import { store } from '@/registry/store'
```

Comment:
we can use the src/common/auth store now

Reply:
I've changed this for the document storage also